### PR TITLE
Added stereo 3D streaming support

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

**Comment by eile** [comment]
Imo needs a spec.md

**Comment by rdumusc** [comment]
Right, should be documented better, working on it.

**Comment by rdumusc** [comment]
Done, see: https://github.com/rdumusc/Deflect/blob/752bd2b5b2f6416cc597362f02d3775cd4fc1226/doc/StereoStreaming.md

File: doc/StereoStreaming.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    deflect::ImageWrapper leftImage( data, width, height, deflect::RGBA );
+    leftImage.view = deflect::View::LEFT_EYE;
+    deflectStream->send( leftImage ) && deflectStream->finishFrame();
+
+    /** ...render right image... */
+
+    deflect::ImageWrapper rightImage( data, width, height, deflect::RGBA );
+    rightImage.view = deflect::View::RIGHT_EYE;
+    deflectStream->send( rightImage ) && deflectStream->finishFrame();
```
Comment (eile) [inline doc/StereoStreaming.md]: I would expect only one finishFrame finishing all eye passes.

Comment (rdumusc) [reply doc/StereoStreaming.md]: OK, this could also be supported without side effects I think. I can give it a try.

Comment (rdumusc) [reply doc/StereoStreaming.md]: implemented in CR#1

**Review by dnachbaur — APPROVED** [review]

**Pull request merged**